site/content/en/docs/contributing/development-environment.md

```diff
@@ -12,7 +12,20 @@ description: 'Installing a development environment for different operating syste
   Ubuntu 22.04/20.04
 
   ```bash
-  sudo apt-get update && sudo apt-get --no-install-recommends install -y build-essential curl git redis-server python3-dev python3-pip python3-venv python3-tk libldap2-dev libsasl2-dev
+    sudo apt-get update && sudo apt-get --no-install-recommends install -y \
+    build-essential \
+    curl \
+    git \
+    redis-server \
+    python3-dev \
+    python3-pip \
+    python3-venv \
+    python3-tk \
+    libldap2-dev \
+    libsasl2-dev \
+    libxml2 \
+    libxmlsec1 \
+    libxmlsec1-openssl
```

Comment:
This seems wrong, because I don't think there's any circumstance under which only the runtime packages would be required - either you're installing lxml/xmlsec from binaries, in which case they come with their own runtime, or you're installing them from source, in which case you need the `-dev` packages too.